src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1282,9 +1294,9 @@ static void addBsonFilter
   }
   else
   {
-    std::string details = std::string("unknown query operator: ") + opr;
+    std::string details = std::string("unknown query operator: '") + opr + "'";
     alarmMgr.badInput(clientIp, details);
-    pushBackFilter = false;
+    return false;
   }
 
   if (pushBackFilter)
```

Comment:
I think there aren't more instances. If that is correct, please go ahead with the fix.